manual_payments.R

```diff
@@ -66,20 +69,25 @@ clean_gs_data <- function(df) {
                   status)
 
   
-  # convert the amount to numeric
+  # convert the dollar amount to numbers
   df$invoice_amount <- as.numeric(gsub('[$,]', '', df$invoice_amount))
   df$discount_amount <- as.numeric(gsub('[$,]', '', df$discount_amount))
   df$total_amount_paid <- as.numeric(gsub('[$,]', '', df$total_amount_paid))
 
-  # update plan_id
+  # update the plan id and set calculated_at time
   df <- df %>%
-    mutate(plan_id = gsub(",", "", plan_id))
+    mutate(plan_id = gsub(",", "", plan_id),
+           calculated_at = Sys.time(),
```

Comment:
@jwinternheimer I think this would return a timestamp using the UTC timezone (looking at [this documentation](https://www.rdocumentation.org/packages/base/versions/3.5.3/topics/Sys.time) ), is that correct? 

I think that is perfect, but I just wanted to confirm with you

Reply:
Good question! It returns a timestamp with the system's timezone. We can make sure that it's UTC by including something like `Sys.setenv(TZ = 'UTC')`.

Reply:
Set the system timezone here: https://github.com/bufferapp/manual-payments/pull/8/commits/86fa1ded18b8b34c896490be35c1cee430fecd4e